File: Module1/8-8/OverridingEx.java
package com.a88;

class Bank1
{
	int rate()
	{
		return 0;
	}
}
class Sbi extends Bank1
{
	int rate()
	{
		return 7;
	}
}
class Icici extends Bank1
{
	int rate()
	{
		return 8;
	}
}
class Axis extends Bank1
{
	int rate()
	{
		return 9;
	}
}

public class OverridingEx 
{
	public static void main(String[] args) 
	{
		
			Bank1 b;
			
			b=new Sbi();
			System.out.println(b.rate());
			
			b=new Icici();
			System.out.println(b.rate());
			
			b=new Axis();
			System.out.println(b.rate());
	}
}

File: Module1/Kotlin/26-8/OverridingEx.kt
open class Bank2
{
 open fun rate():Int
 {
    return  0
 }
}
class Sbi :Bank2()
{
    override fun rate():Int
    {
        return 7
    }
}
class Icici :Bank2()
{
    override fun rate():Int
    {
        return 8
    }
}
class Axis :Bank2()
{
    override fun rate():Int
    {
        return 9
    }
}


fun main()
{
    var b=Bank2()

    b=Sbi()
    println(b.rate())
    b=Icici()
    println(b.rate())
    b=Axis()
    println(b.rate())

}